Src/ReleaseHistory.md

```diff
@@ -13,6 +13,9 @@
 - Rename `SEC101/050.IdentifiableNpmLegacyAuthorToken` to `SEC101/050.NpmIdentifiableAuthorToken` [#683](https://github.com/microsoft/sarif-pattern-matcher/pull/683)
 - Rename `SEC101/017.NpmAuthorToken` to `SEC101/017.NpmAuthorToken` [#683](https://github.com/microsoft/sarif-pattern-matcher/pull/683)
 - Rename `SEC101/006.GitHubPat` to `SEC101/006.GitHubLegacyPat` [#678](https://github.com/microsoft/sarif-pattern-matcher/pull/678)
+- Disable `SEC101/029.AlibabaCloudCredentials` which throws ScanErrors with message: 
+	"ValidationError:Could not load file or assembly 'AlibabaCloud.OpenApiClient, Version=0.1.4.0, Culture=neutral, PublicKeyToken=null' or one of its dependencies. A strongly-named assembly is required. (Exception from HRESULT: 0x80131044)" 
```

Comment:
>ValidationError [](http://example.com/codeflow?start=2&length=15)

What does this look like in the markdown view, did you check? I probably would have tried a block quote here, i.e., prefix the message with a >.

Reply:
I already merged, but it just looks like text in double quotes. That is a good idea though.